libs/config_parser.py

```diff
@@ -66,3 +67,28 @@ def data_pre_processing_for_verifier_image(test_config_dict, end_test_key_str):
             shutil.copytree(test_config_dict["ssl_path"], VERIFIER_SERVICE_PATH + "/ssl")
     if "bash/bash" in test_config_dict['docker_image']:
         shutil.copytree("data/bash", CURATED_APPS_PATH + "/bash")
+        if utils.check_machine() == "Azure Linux Agent":
+            bash_azure_setup()
+
+def bash_azure_setup():
+    bash_dir = os.path.join(CURATED_APPS_PATH, "bash")
+
+    # Updating dockerfile for bash
+    fd = open(os.path.join(bash_dir, "Dockerfile"))
+    fd_contents = fd.read()
+    fd.close()
+    new_data = re.sub('ENV http_proxy "http://proxy-dmz.intel.com:911"\nENV https_proxy "http://proxy-dmz.intel.com:912"\n',
```

Comment:
**re.sub** function is used in conftest.py as well. Can we have a common function in utils.py?

Reply:
Done